vslib/inc/SwitchBCM81724.h

```diff
@@ -99,6 +99,6 @@ namespace saivs
 
             virtual sai_status_t refresh_read_only( _In_ const sai_attr_metadata_t *meta, _In_ sai_object_id_t object_id) override;
             virtual sai_status_t set_switch_default_attributes();
-            virtual sai_status_t initialize_default_objects() override;
+            virtual sai_status_t initialize_default_objects( _In_ uint32_t attr_count, _In_ const sai_attribute_t *attr_list) override;
```

Comment:
one param per line, you can also fix the one in line 100

Reply:
o.k. I'll fix both.

Reply:
Fixed